_sass/_imla-nama.scss

```diff
@@ -231,8 +241,10 @@ article h1 {
 }
 
 article h2::before {
+    font-family: NotoNastaliqUrdu, sans-serif;  //Removed Lalezar due to lack of Urdu numeral support
+    font-weight:bold;
     counter-increment: subhead-count;
-    content: counter(subhead-count);
+    content: counter(subhead-count, persian);
     color: #888;
     margin-left: 0.75em;
 }
```

Comment:
One subtle thing to note here is that the heading is set in Lalezar, while the counter is set in Noto. Both have varying vertical heights; Noto's is taller, which adds some vertical space to the whole heading. To remove that space, we need `line-height: 1` here, which will decrease the line-height of the counter set in Noto, and thus get rid of the additional space.

Reply:
Done in [7776e60](https://github.com/urduweb/imla/pull/17/commits/7776e60b877bbb3b393f8947bbf9fc499dec75fa).